### PR TITLE
Clarify "Areas of focus" in `People/Global_Moderation_Team`

### PR DESCRIPTION
**Pull request by MChecaH**
This new revision [scraps the previous PR](https://github.com/ppy/osu-wiki/pull/11575). Retains the current table format, mentions how all GMTs are allowed (and do) all tasks regardless of area of focus, adds Blushing, and tweaks the entries of team members who have asked for it.

[Discussion in Internal](https://discord.com/channels/90072389919997952/1251185977377034342/1251185987720056914)

[--> PREVIEW <--](https://osu1.roanh.dev/wiki/en/People/Global_Moderation_Team)

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The GMT has seen the new revision.

### CODE REVIEW

**Comment by cl8n** [comment]
> Clayton script working

doesn't look like anything in this PR would need updates

File: wiki/People/Global_Moderation_Team/en.md [hunk 1]
```diff
@@ -70,19 +70,21 @@ Once a Global Moderator chooses to depart from the team, they may be granted the
 
 The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all of the team members. In addition to areas mentioned below, all GMT members take part in [reviewing visual and aural content included in beatmaps](/wiki/Rules/Visual_content_considerations#getting-your-image-assessed).
 
+**A member of the GMT can engage with any tasks within any category that they choose, despite their listed primary responsibilities.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation, or vice versa.
```
Comment (cl8n) [inline wiki/People/Global_Moderation_Team/en.md:73]:
```suggestion
**A member of the GMT may engage with any task within any category that they choose, regardless of their listed area of focus.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation.
```

File: wiki/People/Global_Moderation_Team/en.md [hunk 2]
```diff
@@ -70,19 +70,21 @@ Once a Global Moderator chooses to depart from the team, they may be granted the
...
 | ::{ flag=MY }:: [\_Kobii](https://osu.ppy.sh/users/6209713) | Chinese, Malay, Cantonese, some Japanese | Mapping/modding community moderation |
 | ::{ flag=DE }:: [- Felix](https://osu.ppy.sh/users/8503985) | German | Chat moderation |
-| ::{ flag=HU }:: [[ Another ]](https://osu.ppy.sh/users/3416573) | Hungarian | Chat moderation |
 | ::{ flag=CH }:: [\[ryuu\]](https://osu.ppy.sh/users/5698467) | Russian | Chat moderation |
 | ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | Serbian |  |
 | ::{ flag=US }:: [abraker](https://osu.ppy.sh/users/4635891) |  | Forum moderation |
 | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595) | Dutch, some German |  |
 | ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) |  | Tournament management[^task-Azer] |
 | ::{ flag=MY }:: [bibitaru](https://osu.ppy.sh/users/4482419) | Chinese, Malay | Chat moderation |
+| ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) |  | Chat moderation, forum moderation |
```
Comment (cl8n) [inline wiki/People/Global_Moderation_Team/en.md:87]: do they moderate `#greek` (for table below) ? jw cuz I saw them contribute something to greek on the wiki before.

File: wiki/People/Global_Moderation_Team/en.md [hunk 3]
```diff
@@ -70,19 +70,21 @@ Once a Global Moderator chooses to depart from the team, they may be granted the
 
 The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all of the team members. In addition to areas mentioned below, all GMT members take part in [reviewing visual and aural content included in beatmaps](/wiki/Rules/Visual_content_considerations#getting-your-image-assessed).
 
+**A member of the GMT can engage with any tasks within any category that they choose, despite their listed primary responsibilities.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation, or vice versa.
+
 <!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->
 
 | Name | Additional languages | Area of focus |
```
Comment (cl8n) [inline wiki/People/Global_Moderation_Team/en.md:77]: Petal missing from this table

**Review by cl8n — CHANGES_REQUESTED** [review]

File: wiki/People/Global_Moderation_Team/en.md (hunk 1 above)
Comment (MChecaH) [reply wiki/People/Global_Moderation_Team/en.md:73]: Same wording used as in the [NAT Article](https://osu.ppy.sh/wiki/en/People/Nomination_Assessment_Team#team-members:~:text=A%20member%20of%20the%20NAT%20can%20engage%20with%20any%20tasks%20within%20any%20category%20that%20they%20choose%2C%20despite%20their%20listed%20primary%20responsibilities.%20For%20example%2C%20an%20NAT%20member%20primarily%20responsible%20for%20evaluations%20may%20also%20help%20with%20structural%2Drelated%20tasks%2C%20or%20vice%20versa.). If one is slightly modified, may as well do the other one.

File: wiki/People/Global_Moderation_Team/en.md (hunk 2 above)
Comment (MChecaH) [reply wiki/People/Global_Moderation_Team/en.md:87]: Was not given any additional languages when I asked, so we assume not.

File: wiki/People/Global_Moderation_Team/en.md (hunk 3 above)
Comment (MChecaH) [reply wiki/People/Global_Moderation_Team/en.md:77]: Asking for their area of focus, then adding them.

File: wiki/People/Global_Moderation_Team/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/Global_Moderation_Team/en.md:73]: can be done in this same pr

Comment (MChecaH) [reply wiki/People/Global_Moderation_Team/en.md:73]: Will do then

File: wiki/People/Global_Moderation_Team/en.md (hunk 3 above)
Comment (MChecaH) [reply wiki/People/Global_Moderation_Team/en.md:77]: They wanted it empty. Added as such.

**Review by cl8n — DISMISSED** [review]

**Review by Hiviexd — DISMISSED** [review]

**Comment by cl8n** [comment]
"general moderation" doesn't seem like an "area of focus" :thinking: 
just empty cell is fine.

**Comment by MChecaH** [comment]
The team argued otherwise, wanted to make it clear that despite not having a specific area that they did a bit of everything.

**Comment by cl8n** [comment]
that seems basically opposite of what "focus" means but alright.

**Review by cl8n — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**